internal/ethapi/api.go

```diff
@@ -1872,15 +1869,14 @@ type SignTransactionResult struct {
 // The node needs to have the private key of the account corresponding with
 // the given from address and it needs to be unlocked.
 func (s *TransactionAPI) SignTransaction(ctx context.Context, args TransactionArgs) (*SignTransactionResult, error) {
+	args.blobSidecarAllowed = true
+
 	if args.Gas == nil {
 		return nil, errors.New("gas not specified")
 	}
 	if args.GasPrice == nil && (args.MaxPriorityFeePerGas == nil || args.MaxFeePerGas == nil) {
 		return nil, errors.New("missing gasPrice or maxFeePerGas/maxPriorityFeePerGas")
 	}
-	if args.IsEIP4844() {
```

Comment:
Looks like geth does not support `eth_sendTransaction` with signature (e.g., a JSON string with valid `v`, `r`, `s`, `yParity` fields), it would be useful to let the node relay signed transactions without running a local node with a signer (of course, it can also be achieved using `eth_sendRawTransaction`, but curl command of `eth_sendTransaction` is more readable). Any considerations here?